_posts/0000-01-02-YOUR-USERNAME.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
```

Comment:
---
Title: How to create table in GitHub pages 
Description: using three hyphens continuously creates a box in GitHub page and for columns add as column name with the colon.
Ex: 
Title: Column data
---